PullRequestCreated/PullRequestCreated.ts

```diff
@@ -16,6 +22,25 @@ class PullRequestCreated extends OctokitAction {
   private shouldCreateIssue(pr: PullRequest): boolean {
     return pr != null && !JIRA_ISSUE_PATTERN.test(pr.title);
   }
+
+  private async newIssueTypeAndParent(pr: PullRequest): Promise<Issue> {
+    const mentionnedIssues = this.findMentionnedIssues(pr);
+    console.log(`Found ${mentionnedIssues.length} mentionned issues: ${mentionnedIssues}`);
```

Comment:
The count is redundant
```suggestion
    console.log(`Found mentioned issues: ${mentionnedIssues}`);
```